worlds/wargroove2/Rules.py

```diff
@@ -0,0 +1,28 @@
+from typing import List
+
+from BaseClasses import Region, Location
+from .Levels import Wargroove2Level
+from worlds.AutoWorld import LogicMixin
+
+
+class Wargroove2Logic(LogicMixin):
+    pass
+
+
+def set_rules(multiworld, level_list: [Wargroove2Level],
+              first_level: Wargroove2Level,
+              final_levels: [Wargroove2Level],
+              player: int):
```

Comment:
```suggestion
def set_rules(multiworld, level_list: [Wargroove2Level],
              first_level: Wargroove2Level,
              final_levels: [Wargroove2Level],
              player: int) -> None:
```
Another candidate for swapping to just loading in `world`, though this one is probably more fine as-is?